src/main/cms/cms_menu_vtx_tramp.c

```diff
@@ -148,7 +148,7 @@ static long trampCmsCommence(displayPort_t *pDisp, const void *self)
     UNUSED(self);
 
     trampSetBandAndChannel(trampCmsBand, trampCmsChan);
-    trampSetRFPower(trampPowerTable[trampCmsPower-1]);
+    vtxTrampSetPowerByIndex(trampCmsPower);
 
```

Comment:
Rather than calling these two functions, I found it more efficient to call this:
```c++
   vtxTrampSetBandChanPowerByIndex(trampCmsBand, trampCmsChan, trampCmsPower);
```

Reply:
Adding another function to the vtx vtable would require a rework in all three VTX implementations when this issue is isolated to one line of code.  Alternatively, simply creating a wrapper would just end up running these two functions as seen above.  Given that this issue is isolated to this one line of code, making a change of this type might be over-engineering it a bit.  Not a bad idea, nonetheless, but can we see if this fixes the issue at hand until a more robust solution can be built?